bin/docker-push

```diff
@@ -14,6 +14,6 @@ bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 # shellcheck source=_docker.sh
 . "$bindir"/_docker.sh
 
-for img in cli-bin cni-plugin controller metrics-api debug grafana proxy web ; do
+for img in cli-bin cni-plugin controller metrics-api debug grafana proxy web tap; do
```

Comment:
I can add it. I think we've gotten by since this script is used a lot less now after the previous CI changes.

In general, there are quite a few outdated scripts that probably don't need to be around anymore, but that's outside of this change.